File: src/main/java/io/github/mg138/tsbook/config/gui/element/ArmorElementSetting.java
package io.github.mg138.tsbook.config.gui.element;

import org.bukkit.configuration.ConfigurationSection;

public class ArmorElementSetting {
    public final String type;

    public ArmorElementSetting(ConfigurationSection section) {
        this.type = section.getString("TYPE");
    }
}

File: src/main/java/io/github/mg138/tsbook/setting/gui/armor/element/ArmorElementSetting.kt
package io.github.mg138.tsbook.setting.gui.armor.element

import io.github.mg138.tsbook.setting.gui.element.GUIElementSetting
import org.bukkit.Material

class ArmorElementSetting(
    slot: Int,
    material: Material,
    count: Int,
    name: String,
    lore: List<String>,
    model: Int,
    val setting: ArmorSetting
) : GUIElementSetting(slot, material, count, name, lore, model)